src/jenkins/BuildManifest.groovy

```diff
@@ -49,6 +68,14 @@ class BuildManifest implements Serializable {
             this.getArtifactRoot(jobName, buildNumber)
         ].join('/')
     }
-}
-
 
+    public String getPackageName() {
+        String packagePrefix = [
+            this.buid.getFilename(),
```

Comment:
Typo here.

since this gets all properties from build, the method should be implemented on build as well